spanner/tests/unit/test_keyset.py

```diff
@@ -30,49 +33,68 @@ def test_ctor_no_start_no_end(self):
         with self.assertRaises(ValueError):
             self._make_one()
 
-    def test_ctor_w_start_open_and_start_closed(self):
+    def test_ctor_start_open_and_start_closed(self):
```

Comment:
FWIW, your editorial removal of the `w_` (short for "with") hasn't made these names easier to read (it hasn't made them harder to read either, but why bother with the change?)

Reply:
it shortened some tests just enough to fit the name I think was my reasoning.  I wanted to make it consistent across the file.